apps/opik-frontend/src/components/pages-shared/llm/SetupProviderDialog/ProviderGrid.tsx

```diff
@@ -13,11 +13,20 @@ const ProviderGrid: React.FC<ProviderGridProps> = ({
   selectedProvider,
   onSelectProvider,
 }) => {
+  // Filter out OPIK_BUILTIN - it's a system provider that users don't configure
+  const configurableProviders = useMemo(
+    () =>
+      PROVIDERS_OPTIONS.filter(
+        (provider) => provider.value !== PROVIDER_TYPE.OPIK_BUILTIN,
+      ),
+    [],
+  );
```

Comment:
We don't have this OPIK Provider here on a modal when you open playground 

  <img width="1852" height="882" alt="image" src="https://github.com/user-attachments/assets/f1f8f8ee-04c0-4244-8e49-b1637812f071" />


I see that it's on purpose but we still don't see it as configured in a select
<img width="1855" height="884" alt="image" src="https://github.com/user-attachments/assets/09eaf4c3-b800-4d97-82fa-8ffb66986243" />